tests/rados/test_track_availibility_status.py

```diff
@@ -0,0 +1,604 @@
+        )
+
+        log.info(
+            "==== Case 2: Verification of the Mean Time To Recover(MTTR) value ===="
+        )
+
+        result_MTTR = test_MTTR_value(client, pool_name)
+
+        if not result_MTTR:
+            log.error("Case 2: Verification of the MTTR failed ")
+
+        log.info(
+            "==== Case 2: Verification of the Mean Time To Recover(MTTR) value is completed ===="
+        )
+
+        log.info("==== Case 3: Verification of the score value ====")
+        result_score = test_score_value(client, pool_name)
+        if not result_score:
+            log.error("Case 3: Verification of the score value failed ")
+        log.info("==== Case 3: Verification of the score value is completed ====")
+
+        log.info(
+            "==== Case 4: Verification of the score value displayed on multiple pools ===="
+        )
+        previous_pool_data = get_pool_availability_status(client)
+        previous_total_pool = len(previous_pool_data)
+        msg_pre_pool = (
+            f"The number of  pools before starting tests are - {previous_total_pool}"
+        )
+        log.info(msg_pre_pool)
+        new_pool_names = []
+        for pool_num in range(20):
+            rados_object.create_pool(pool_name=f"test_pool{pool_num}")
+            new_pool_names.append(f"test_pool{pool_num}")
+
+        after_pool_data = get_pool_availability_status(client)
+        after_total_pool = len(after_pool_data)
+        msg_after_pool = (
+            f" The total number of pools after creating are - {after_total_pool}"
+        )
+        log.info(msg_after_pool)
+        if (after_total_pool - previous_total_pool) != 20:
+            log.error("The total pool data is not displayed")
+            return 1
+
+        random_pool = random.choice(new_pool_names)
+        msg_random_pool = f"The random pool selected is - {random_pool}"
+        log.info(msg_random_pool)
+        result_all_tests = test_all_values(client, random_pool)
+        if not result_all_tests:
+            msg_error = f" The track availability tests are failed on {random_pool}"
+            log.error(msg_error)
+            return 1
+
+        msg_pool_deletion = f" The {random_pool} is selected to delete the pool "
+        log.info(msg_pool_deletion)
+        method_should_succeed(rados_object.delete_pool, random_pool)
+        after_delete_pool_details = get_pool_availability_status(client)
+        if random_pool in after_delete_pool_details:
+            msg_error = f" The {random_pool} pool details exists in the availability-status output"
+            log.error(msg_error)
+            return 1
+        msg_info = (
+            f" The {random_pool} pool details not exists in the availability-status output."
+            f"The pool available status output is -{after_delete_pool_details}"
+        )
+        log.info(msg_info)
+        log.info("Deleting the created pools from the cluster")
+        for tmp_pool_name in new_pool_names:
```

Comment:
remove the name of deleted pool from `new_pool_names` for deletion.

Reply:
Removed the deleted pool from the new_pool_names list.